File: bitlap-core/src/test/kotlin/org/bitlap/core/test/sql/CMDTest.kt
/**
 * Copyright (C) 2023 bitlap.org .
 */
package org.bitlap.core.test.sql

import io.kotest.matchers.shouldBe
import org.bitlap.core.test.base.BaseLocalFsTest
import org.bitlap.core.test.base.SqlChecker

/**
 * Mail: chk19940609@gmail.com
 * Created by IceMimosa
 * Date: 2021/9/14
 */
class CMDTest : BaseLocalFsTest(), SqlChecker {

    init {
        "test cmd sql" {
            sql("run example 'xxx'") shouldBe listOf(listOf("hello 'xxx'"))
        }
    }
}

File: bitlap-core/src/test/scala/org/bitlap/core/test/sql/CMDTest.scala
/**
 * Copyright (C) 2023 bitlap.org .
 */
package org.bitlap.core.test.sql

import org.bitlap.core.test.base.BaseLocalFsTest
import org.bitlap.core.test.base.SqlChecker

class CMDTest extends BaseLocalFsTest with SqlChecker {

  test("test cmd sql") {
    checkRows(
      s"run example 'xxx'",
      List(List(s"hello 'xxx'"))
    )
  }
}
